README.md

```diff
@@ -1,55 +1,82 @@
+
 A service to get the user's current location, based on Geolocator, suited to UWP or [Uno](https://platform.uno/) apps.
 
 GeolocatorService aims to simplify getting the user's current location and handling most common scenarios, including getting the permission to obtain this location and handling cases where this permission is denied.
 
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
-
 ## Getting Started
 
-#### Xamarin / UWP
-Add the GeolocatorService nuget package to your project heads.
-
-To have access to reactive extensions such as `GetAndObserveLocation`, add the GeolocatorService.Reactive nuget package to your project.
-
-#### .Net Mobile / WinUI
-
-Add the GeolocatorService.Uno.WinUI nuget package to your project heads.
-
-To have access to reactive extensions such as `GetAndObserveLocation`, add the GeolocatorService.Reactive.Uno.WinUI nuget package to your project.
-
-### Add the relevant permissions
-
-#### UWP
-
-Add the Location capability in your manifest.
-
-#### iOS
-
-Add the NSLocationWhenInUsageDescription and NSLocationUsageDescription values to your info.plist file. This is the message that is displayed to your user when the permission for their location is requested. For instance,
-
-```
-<key>NSLocationWhenInUseUsageDescription</key>
-<string>TODO NSLocationWhenInUsageDescription</string>
-<key>NSLocationUsageDescription</key>
-<string>TODO NSLocationUsageDescription</string>
-```
-
-#### Android
-
-Add the location permission to your AssemblyInfo.cs file.
-
-```
-[assembly: UsesPermission(Android.Manifest.Permission.AccessFineLocation)]
-```
+1. ### Add Package
+
+   #### .Net Mobile / WinUI
+   
+   Add the GeolocatorService.Uno.WinUI nuget package to your project heads.
+   
+   To have access to reactive extensions such as `GetAndObserveLocation`, add the GeolocatorService.Reactive.Uno.WinUI nuget package to your project.
+   
+   #### Xamarin / UWP
+   Add the GeolocatorService nuget package to your project heads.
+   
+   To have access to reactive extensions such as `GetAndObserveLocation`, add the GeolocatorService.Reactive nuget package to your project.
+
+1. ### Add the relevant permissions
+
+   #### Windows
+   
+   Add the Location capability in your manifest.
+
+   ``` xml
+    <!-- Package.appxmanifest -->
+    <Package ...>
+        <!-- (...) -->
+        <Capabilities>
+            <DeviceCapability Name="location"/>
+        </Capabilities>
+    </Package>
+   ```
+    <details>
+        <summary>Visual Studio appxmanisfest UI</summary>
+
+    ![appxmanifest](./doc/img/appxmanisfest.png)
+    </details>
+   
+   #### iOS
+   
+   Include the NSLocationWhenInUsageDescription and NSLocationUsageDescription entries in your info.plist file. These values define the message shown to users when they are asked for location permission. For instance,
```

Comment:
format `NSLocationWhenInUsageDescription` and `NSLocationUsageDescription` as code (like I just did in this comment) to improve readability.